test/functional/fixtures/api/es-next/navigate-to-and-test-page/testcafe-fixtures/navigate-to-test.js

```diff
@@ -15,3 +20,16 @@ test
 ('Page directive', async t => {
     await t.click('#button');
 });
+
+test
```

Comment:
I can't see any tests for absolute `file://` URLs. We need to test `navigateTo` action as well